pkg/ucp/frontend/controller/resourcegroups/util_test.go

```diff
@@ -107,138 +107,138 @@ func Test_ValidateDownstream(t *testing.T) {
 			},
 		}
 
-		mock := setup(t)
-		mock.EXPECT().Get(gomock.Any(), id.PlaneScope()).Return(&database.Object{Data: plane}, nil).Times(1)
-		mock.EXPECT().Get(gomock.Any(), id.RootScope()).Return(&database.Object{Data: resourceGroup}, nil).Times(1)
-		mock.EXPECT().Get(gomock.Any(), resourceTypeResource.ID).Return(&database.Object{Data: resourceTypeResource}, nil).Times(1)
-		mock.EXPECT().Get(gomock.Any(), locationResource.ID).Return(&database.Object{Data: locationResource}, nil).Times(1)
+		databaseClient := setup(t)
```

Comment:
This was just a rename to make things clearer. It's optional so I could revert if necessary.